src/compile/facet.ts

```diff
@@ -166,6 +166,10 @@ export class FacetModel extends ModelWithField {
     return assembleScalesForModel(this);
   }
 
+  public assembleProjections(): VgProjection[] {
```

Comment:
Let's get rid of this method -- just like https://github.com/vega/vega-lite/pull/2889